docs/how-to-guides/eagleye-integration-guide.md

```diff
@@ -0,0 +1,115 @@
+
+## Eagleye setup
+
+### GNSS ROS driver setting
+
+In addition to latitude, longitude, and height information (`sensor_msgs/msg/NavSatFix`), Eagleye also requires velocity information from GNSS.
+The settings needed for each GNSS ROS driver are as follows:
+
+- [ublox_gps](https://github.com/KumarRobotics/ublox/tree/ros2/ublox_gps): This ROS driver publishes `sensor_msgs/msg/NavSatFix` and `geometry_msgs/msg/TwistWithCovarianceStamped` required by Eagleye with default settings. Therefore, no additional settings are required.
+- [septentrio_gnss_driver](https://github.com/septentrio-gnss/septentrio_gnss_driver/tree/ros2): Set `publish.navsatfix` and `publish.twist` in the config file [`gnss.yaml`](https://github.com/septentrio-gnss/septentrio_gnss_driver/blob/ros2/config/gnss.yaml#L90) to `true`
+
+### Eagleye topic setting
+
+You must correctly specify input topics for GNSS latitude, longitude, and height information, GNSS speed information, IMU information, and vehicle speed information in the [`eagleye_config.yaml`](https://github.com/MapIV/autoware_launch/blob/3f04a9dd7bc4a4c49d4ec790e3f6b9958ab822da/autoware_launch/config/localization/eagleye_config.param.yaml#L7-L16).
+
+```yaml
+# Topic
+twist:
+  twist_type: 1 # TwistStamped : 0, TwistWithCovarianceStamped: 1
+  twist_topic: /sensing/vehicle_velocity_converter/twist_with_covariance
+imu_topic: /sensing/imu/tamagawa/imu_raw
+gnss:
+  velocity_source_type: 2 # rtklib_msgs/RtklibNav: 0, nmea_msgs/Sentence: 1, ublox_msgs/NavPVT: 2, geometry_msgs/TwistWithCovarianceStamped: 3
+  velocity_source_topic: /sensing/gnss/ublox/navpvt
+  llh_source_type: 2 # rtklib_msgs/RtklibNav: 0, nmea_msgs/Sentence: 1, sensor_msgs/NavSatFix: 2
+  llh_source_topic: /sensing/gnss/ublox/nav_sat_fix
+```
+
+Also, the frequency of GNSS and IMU must be set in [`eagleye_config.yaml`](https://github.com/MapIV/autoware_launch/blob/3f04a9dd7bc4a4c49d4ec790e3f6b9958ab822da/autoware_launch/config/localization/eagleye_config.param.yaml#L36)
+
+```yaml
+common:
+  imu_rate: 50
+  gnss_rate: 5
+```
+
+### Eagleye parameter tuning
+
+The basic parameters that do not need to be changed except those mentioned above, i.e., topic names and sensors' frequency, are described below [here](https://github.com/MapIV/eagleye/tree/autoware-main/eagleye_rt/config).
+Additionally, the parameters for converting NavSatFix to pose is listed in [`fix2pose.yaml`](https://github.com/MapIV/eagleye/blob/autoware-main/eagleye_util/fix2pose/launch/fix2pose.xml).
+
+### Autoware setting for Eagleye
+
+Please refer to [this PR](https://github.com/autowarefoundation/autoware/pull/3261) when introducing Eagleye to your Autoware setup.
```

Comment:
If you have necessary steps in order to setup EagleEye (e.g., adding cloning new repositories), I would like to have them written them down here rather than putting links to PR since PR's purpose is not for writing instructions.

Reply:
Fixed